mhkit/wave/io/ndbc.py

```diff
@@ -371,7 +385,10 @@ def request_data(parameter, filenames, proxy=None, clear_cache=False, to_pandas=
             # Create a unique filename based on the function parameters for caching
             hash_params = f"{buoy_id}_{parameter}_{year}_{filename}"
             cached_data, _, _ = handle_caching(
-                hash_params, cache_dir, clear_cache_file=clear_cache
+                hash_params,
+                cache_dir,
+                cache_content={"data": None, "metadata": None, "write_json": None},
+                clear_cache_file=clear_cache,
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args